content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -290,5 +290,32 @@ file:///Users/dvc/example-get-started/plots.html
 > [Git revisions](https://git-scm.com/docs/gitrevisions) (commits, tags, branch
 > names) to compare.
 
+## Badges
```

Comment:
Tbh, this a bit too much for get-started. Let's put it a separate section / page in the UG's experiments?

Reply:
and we can put a bit more example with DVC logo, etc

Reply:
Do you mean somewhere under https://dvc.org/doc/user-guide or under https://dvc.org/doc/user-guide/experiment-management? The former seems much more appropriate.

Reply:
> The former seems much more appropriate as badges are for pipeline metrics, not experiment metrics.

they are more or less the same for me (and from the end user perspective I hope)

> Do you mean somewhere under https://dvc.org/doc/user-guide or under https://dvc.org/doc/user-guide/experiment-management

this https://dvc.org/doc/user-guide/experiment-management probably ... or when we have something like Best Practices ... we can mention it in a few places ... we also have https://dvc.org/doc/install/plugins - not directly related, but might make sense to mention there

Reply:
> they are more or less the same for me (and from the end user perspective I hope)

Very confused by this. I thought:

1. `dvc {stage,repro}`: [pipeline](https://dvc.org/doc/start/data-management/pipelines) DAG similar to makefiles
2. `dvc exp`: ephemeral [experimentation](https://dvc.org/doc/start/experiments)

are two completely and utterly different things? vis. https://github.com/iterative/dvc.org/issues/3630

Reply:
Great info but I also agree on not putting it on Get Started. I also don't think we need this in the cmd ref as it doesn't have much to do with the metrics command usage.

Reply:
> `dvc exp` is just another way to run and capture results

but it won't show up in `metrics.json`. That requires `exp run && exp apply && git commit`... aka `dvc repro && git commit`, right?

Reply:
It requires `dvc exp` and `git commit` - similar to `dvc repro` as far as I know.

it's fine to mention about this here, but the whole section is too much ... we can put the details / instruction to plugins and keep some admonition here, wdyt?